### DISCUSSION TITLE
aws_s3 sink, azure to aws access issue...different account than where my role and external_id located

### DISCUSSION THREAD

**Discussion by trax721** [vectordotdev/vector#21834] [Q&A]
I am having an issue with an aws_s3 sink. I am able to send logs to an s3 bucket in my account, but there is some issue with the profile that is in another account. is there a particular setting i need to do for this?

**Comment by pront** [comment]
Hello, are you perhaps referring to the [authentication profile](https://vector.dev/docs/reference/configuration/sinks/aws_s3/#auth.profile)?
